src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ProductVariantTranslation.xml

```diff
@@ -12,12 +12,12 @@
         <itemOperations>
             <itemOperation name="admin_get">
                 <attribute name="method">GET</attribute>
-                <attribute name="path">/admin/product-variant-translation/{id}</attribute>
```

Comment:
This change should be probably noted in UPGRADE-API file

Reply:
https://github.com/Sylius/Sylius/pull/14837/files#diff-b907ff375a390e4387156c574995b944b98b5e66d47841cd85cf020cc0a340dc